cds_ils/patrons/serializers.py

```diff
@@ -15,8 +15,14 @@ def serialize_on_loan_literature_info(loan):
     """Serialize loan information."""
     Document = current_app_ils.document_record_cls
     document = Document.get_record_by_pid(loan["document_pid"])
+    if "barcode" in loan["item"]:
```

Comment:
This is ok totally ok! If you want to do it in a more pythonic way:
```
barcode = loan["item"].get("barcode", "")
```

Reply:
Tried that, but was getting this error:
`AttributeError: 'AttrDict' object has no attribute 'get'`